src/compose.ts

```diff
@@ -37,11 +37,11 @@ export function defaultComposition(
 			context = 'build: "."';
 		}
 	}
-	return `# Auto-generated compose file by resin-compose-parse@v${packageVersion}
+	return `# Auto-generated compose file by balena-compose-parse@v${packageVersion}
 version: '2.1'
 networks: {}
 volumes:
-  resin-data: {}
+  balena-data: {}
```

Comment:
Is there anything special about `resin-data` as a named volume? The docs website still documents `resin-data`, and I always assumed there was a reason for that:
https://www.balena.io/docs/learn/develop/multicontainer/#named-volumes

Reply:
uups, you're probably right. I just did `%s/resin/balena/g` on the entire project ^^ cc @pipex to make sure...

Reply:
Or not fail necessarily, but they will see their data disappear.

Reply:
@pipex this only applies to apps without a compose file correct? since this ^ is the compose file we generate for them in the background